<launch>
	
	<arg name="controller" default="phobic_mp" doc="Name of the controller to be loaded (e.g. joint_trajectoy_controller)" />
	<!-- <arg name="stopped_controllers" default="OneTaskInverseKinematics" doc="Additional controllers to be loaded, but not started (can be started later with controller_manager)" /> -->

	<!-- Load joint controller configurations from YAML file to parameter server -->
	<rosparam file="$(find desperate_housewife)/config/lwr_controllers.yaml" command="load"/>
	
	<node ns="left_arm" name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="controller"/> 
	<!-- <node ns="left_arm" name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="joint_state_controller $(arg controller)"/>  -->
	
	<!-- <node ns="left_arm" name="controller_spawner_stopped" pkg="controller_manager" type="spawner" respawn="false" output="screen"/>  -->
	
</launch>
